subjects/java/piscine/ListEquals/README.md

```diff
@@ -4,15 +4,15 @@
 
 Create a file `ListEquals.java`.
 
-Write a function `areListEquals` that returns true if the lists as parameters are equals. Returns false otherwise.
+Write a function `areListsEquals` that returns true if the lists as parameters are equals. Returns false otherwise.
```

Comment:
While the intent of the function is clear, there is a grammatical error in the function name. According to grammar rules, the function name should be `areListsEqual` instead of areListsEquals. To maintain consistency with the singular form of "list," the word "equals" should be changed to "equal".

Here's the revised sentence:

"Write a function `areListsEqual` that returns true if the lists as parameters are equal. Returns false otherwise."

The same applies also for `areSetsEqual` and `areMapsEqual`.

This updated version adheres to proper grammar rules and accurately conveys the function's purpose.